src/EditorFeatures/CSharpTest/Completion/CompletionProviders/OverrideCompletionProviderTests.cs

```diff
@@ -775,26 +775,22 @@ public class SomeClass : Base
         }
 
         [WorkItem(529714, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/529714")]
-        [WpfFact(Skip = "529714"), Trait(Traits.Feature, Traits.Features.Completion)]
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
         public async Task GenericMethodTypeParametersRenamed()
```

Comment:
Maybe rename the test to `GenericMethodTypeParametersNotRenamed`, since they aren't getting renamed? Similar for the other test.